libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -632,7 +633,6 @@
       "MeteringPointAdministrator": "Metering point administrator",
       "SystemOperator": "System operator",
       "DanishEnergyAgency": "Energistyrelsen",
-      "ElOverblik": "ElOverblik",
```

Comment:
issue: this translation is also used in the "Market roles" tab under "Actors". So if we really should get rid of it, then we also need to get rid of "EicFunction.ElOverblik" enum property in `.../generated/graphql.ts`

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/235316ca-1bff-4dc9-92b0-2c72acf7e0dc)